mappings/net/minecraft/client/render/debug/StructureDebugRenderer.mapping

```diff
@@ -1,4 +1,11 @@
 CLASS net/minecraft/class_870 net/minecraft/client/render/debug/StructureDebugRenderer
+	FIELD field_32916 RANGE I
 	FIELD field_4624 client Lnet/minecraft/class_310;
+	FIELD field_4626 structureBoundingBoxesByDimension Ljava/util/Map;
+	FIELD field_4627 structurePiecesBoundingBoxesByDimension Ljava/util/Map;
```

Comment:
```suggestion
	FIELD field_4626 structureBoundingBoxes Ljava/util/Map;
	FIELD field_4627 structurePiecesBoundingBoxes Ljava/util/Map;
```
the `byDimension` suffix is extraneous and we don't really need to emphasize it's a per-dimension lookup if this is just the general store of bounding boxes, etc.